documents/build.md

```diff
@@ -10,15 +10,15 @@ description: Learn how to build your own app via LBRY in this comprehensive guid
 Want to build something on top of LBRY? This is the place to get started.
 
 At least, it's the place to get started so long as you have some idea of what LBRY _is_. If you don't, you should read the
-[Overview](/overview) first.
+[Overview](/overview) first and the [spec](/spec) for more technical details. 
```

Comment:
Overview is fine, reading the spec is unnecessary for most people who want to build